src/test/java/it/gov/pagopa/payhub/activities/activity/paymentsreporting/PaymentsReportingIngestionFlowFileActivityImplTest.java

```diff
@@ -33,10 +41,15 @@ class PaymentsReportingIngestionFlowFileActivityImplTest {
 	private IngestionFlowFileRetrieverService ingestionFlowFileRetrieverServiceMock;
 	@Mock
 	private FlussoRiversamentoUnmarshallerService flussoRiversamentoUnmarshallerServiceMock;
+	@Mock
+	private PaymentsReportingIngestionFlowFileValidatorService paymentsReportingIngestionFlowFileValidatorServiceMock;
+	@Mock
+	private PaymentsReportingMapperService paymentsReportingMapperServiceMock;
 	private PaymentsReportingIngestionFlowFileActivityImpl ingestionActivity;
 
 	@TempDir
 	private File tempDir;
+	private CtFlussoRiversamento ctFlussoRiversamento;
```

Comment:
why you have an instance variable, but you are calling the new on each test?

Reply:
It just a typo, Ill' fix it